docs/tutorial.md

```diff
@@ -9,20 +9,23 @@ All InfraKit plugins will by default open the unix socket located at /run/infrak
 exists on your host:
 
 ```shell
-$ mkdir -p /run/infrakit/plugins/
-$ chmod 777 /run/infrakit/plugins
+# Make sure directory exists for plugins to discover each other (default is: /run/infrakit/plugins/)
+$ mkdir -p /$TMPDIR/infrakit/plugins/
 ```
 
 Start the default Group plugin
 
+**_NOTE:_** You can set the listen url as an env variale via `export INFRAKIT_PLUGINS_DIR="/$TMPDIR/infrakit/plugins/"` instead of passing it as a flag. The default socket will be used, unless the `--name` flag is passed, when the env variable is set. 
+
 ```shell
+$ export INFRAKIT_PLUGINS_DIR="/$TMPDIR/infrakit/plugins/"
 $ build/infrakit-group-default --log 5
 INFO[0000] Starting discovery
-DEBU[0000] Opening: /run/infrakit/plugins
+DEBU[0000] Opening: /$TMPDIR/infrakit/plugins
```

Comment:
This is exactly my worry with including console output, rogue find/replace leads to lies!  In reality `$TMPDIR` should be expanded (ditto for numerous other lines).  I'll let you choose how to proceed, i think the console output needs refining anyways, so it's likely to receive another pass soon regardless.